### PR TITLE
chore: modernise the build

### PR DESCRIPTION
**Pull request by dnwe**
- bump travis to go1.13
- switch from dep to go modules
- switch from gometalinter to golangci-lint
- no longer need gotestcover

### CODE REVIEW

**Review by caarlos0 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
 
+export PATH := ./bin:$(PATH)
+export GO111MODULE := on
+export GOPROXY = https://proxy.golang.org,direct
+
+GOBIN?=$(shell go env GOBIN)
+GOLANGCI_LINT := $(GOBIN)/golangci-lint
+$(GOLANGCI_LINT): 
+	cd "$$(mktemp -d)" && \
+		go get github.com/golangci/golangci-lint
```
Comment (caarlos0) [inline Makefile]: maybe use the `tools.go` approach? e.g. https://github.com/goreleaser/goreleaser/blob/master/tools.go

**Review by caarlos0 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (dnwe) [reply Makefile]: @caarlos0 happy to switch to that if you'd prefer?

Comment (caarlos0) [reply Makefile]: if your way its working, its fine by me too :)

Comment (dnwe) [reply Makefile]: ok cool, will merge

**Pull request merged**